week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +50,19 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+for ( let i = 0; i <arr.length;i++){
+  arr[i] = Number(arr[i]).toFixed(2);
+  if (arr [i] > 100 )
+      arr[i]=100;
+  arr [i] =arr [i]+ "%";
+console.log([i]);
+    
+}
+return arr;
+}
```

Comment:
I think this test doesn't pass the test?
For example, whole number should not be rounded to two decimal places.
e.g. 10 should be formatted as `10%` and NOT `10.00%`